From 7d94631991a5d530884664705341a4410ff45f74 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=89rico=20Nogueira?= <erico.erc@gmail.com>
Date: Thu, 29 Jul 2021 15:57:00 -0300
Subject: [PATCH] bar: fix typo.

Running 'yambar --backend=wayland' would always error out saying it was
built without wayland support.
---
 bar/bar.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bar/bar.c b/bar/bar.c
index 4829162..c7b1ea5 100644
--- a/bar/bar.c
+++ b/bar/bar.c
@@ -399,7 +399,7 @@ bar_new(const struct bar_config *config)
         break;
 
     case BAR_BACKEND_WAYLAND:
-#if defined(BAR_WAYLAND)
+#if defined(ENABLE_WAYLAND)
         backend_data = bar_backend_wayland_new();
         backend_iface = &wayland_backend_iface;
 #else
-- 
2.32.0

